README.md

```diff
@@ -31,10 +31,16 @@ Compared to the extract-text-webpack-plugin:
 
 To begin, you'll need to install `mini-css-extract-plugin`:
 
-```bash
+```console
 npm install --save-dev mini-css-extract-plugin
 ```
 
+or
+
+```console
+yarn add -D mini-css-extract-plugin
```

Comment:
Change it on `npm`, because we use npm

Reply:
If we want to provide yarn, we should also provide pnpm example, i.e. for all package managers

Reply:
🤔  I've seen npm and yarn in some repos. That is why I started adding yarn in all repos. Should we consider adding `pnpm` as well?

Reply:
If we add `yarn` we should add `pnpm` too